docs/Tutorial_Matrix_Profile_Top_Ten_short.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
<p>Technically, we don't need the blue curve. We only need <code>A</code> and <code>B</code> to support our point.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/667/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I remove it, and follow your above guide.</p>